### PR TITLE
update jupyterbook toc links to splash pages

### PR DESCRIPTION
**Pull request by JessicaS11**
As I pointed out during our meeting yesterday, the links to the teams and schedule on the splash page weren't showing up in the Jupyterbook. The links weren't pointing specifically to the jupyter-template repo github.io page.

### CODE REVIEW

**Comment by JessicaS11** [comment]
Note: the jupyterbook front page links are hard coded via the `website_url` parameter, so they won't point to the preview-built pages in a PR.

**Comment by JessicaS11** [comment]
@jomey Do we need to specifically create the `#section` tags in our cookiecutter file so they exist on the splash page? If I use `- url: https://uwhackweek.github.io/jupyterbook-template/` in the `_toc.yml` the entries generate fine (but only go to the top of the page), but as soon as I add `#team` (`- url: https://uwhackweek.github.io/jupyterbook-template/#team`) there's something about them jupyterbook doesn't like, so they don't show up in the TOC. There's no warning or error message I can find (even with --verbose) to indicate a problem, and entering the links directly into a browser works fine.

**Comment by jomey** [comment]
> @jomey @scottyhq Any ideas what jupyterbook doesn't like? If I use `- url: https://uwhackweek.github.io/jupyterbook-template/` in the `_toc.yml` the entries generate fine (but only go to the top of the splash page), but as soon as I add `#team` (`- url: https://uwhackweek.github.io/jupyterbook-template/#team`) there's something about them jupyterbook doesn't like, so they don't show up in the TOC. There's no warning or error message I can find (even with --verbose) to indicate a problem, and entering the links directly into a browser works fine. The links to the team and schedule section on the home page of the jupyterbook also work fine, so I'm guessing it's something in jupyterbook's creation of the toc that's causing them to get lost?

Let me have a quick peak locally

**Comment by JessicaS11** [comment]
For anyone watching - we've gotten as far as figuring out that the Jupyterbook toc-url directive doesn't seem to like anchor tags...

**Comment by jomey** [comment]
Couldn't figure out why JupyterBook in combination with the sphinx-external-toc extension was not happy with hashes '#' in TOC urls. Added some simple logic on the main splash site to navigate to a section when requested via URL parameter. Added benefit is that this now behaves identical to clicking on a link in the navigation bar.

Open to reverting this again if this feels as too much custom logic.

**Comment by jomey** [comment]
Note: When testing this, make sure you use the preview URL. Right now we have 
https://uwhackweek.github.io/jupyterbook-template/index.html
in the TOC and this is the test URL:
https://deploy-preview-64--hackweek-template.netlify.app/index.html?jump_to=team

File: book/_toc.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ parts:
...
   chapters:
   - file: application
-  - url: https://uwhackweek.github.io#schedule-section
-    title: Schedule
-  - url: https://uwhackweek.github.io#team
-    title: Team
+  - title: Schedule
+    url: https://uwhackweek.github.io/jupyterbook-template/index.html?jump_to=team
+  - title: Team
+    url: https://uwhackweek.github.io/jupyterbook-template/index.html?jump_to=schedule
```
Comment (JessicaS11) [inline book/_toc.yml:12]:
```suggestion
  - title: Team
    url: https://uwhackweek.github.io/jupyterbook-template/index.html?jump_to=team
  - title: Schedule
    url: https://uwhackweek.github.io/jupyterbook-template/index.html?jump_to=schedule
```

File: {{ cookiecutter.repo_directory }}/assets/js/main.js [hunk 2]
```diff
@@ -137,4 +142,3 @@ function startCountDown(counterDiv) {
     }
 }
 
-startCountDown(counterDiv);
```
Comment (JessicaS11) [inline {{ cookiecutter.repo_directory }}/assets/js/main.js:140]: Does this remove the countdown counter?

**Comment by JessicaS11** [comment]
@jomey This looks like a great workaround to me (see my suggested changes to fix switched links).

> Couldn't figure out why JupyterBook in combination with the sphinx-external-toc extension was not happy with hashes '#' in TOC urls.

Do you think we ought to file a bug report with JupyterBook?

File: {{ cookiecutter.repo_directory }}/assets/js/main.js (hunk 2 above)
Comment (jomey) [reply {{ cookiecutter.repo_directory }}/assets/js/main.js:140]: My bad and overzealous delete key

**Comment by jomey** [comment]
> > Couldn't figure out why JupyterBook in combination with the sphinx-external-toc extension was not happy with hashes '#' in TOC urls.
> 
> Do you think we ought to file a bug report with JupyterBook?

Ultimately yes, I would love to figure out a few more details and then file with the appropriate project. Right now I can’t pinpoint if this is JupyterBook, sphinx-external-toc, or an issue somewhere in between those two.

**Review by scottyhq — APPROVED** [review]
nice solution!

**Pull request merged**